docs/changehistory/NextVersion.md

```diff
@@ -618,7 +618,7 @@ Allow to attach a ECDb/IModel to connection and run ECSQL that combines data fro
 
 Example of attaching a snapshot to a master file and running a query that combines data from both databases:
 ```ts
- const master = SnapshotDb.openFile(masterFile);
+     const master = SnapshotDb.openFile(masterFile);
```

Comment:
Sorry; looks like the other lines are three-space indented, while my suggested change was 4. I think the indent for all lines should be either 4 spaces or 2, but I'm not sure which is better.

Reply:
Never mind; looks like my suggested change somehow had 5 spaces, so just fix that.